lib/src/view/home/home_tab_screen.dart

```diff
@@ -602,16 +602,21 @@ class _PlayerScreenButton extends ConsumerWidget {
 
   @override
   Widget build(BuildContext context, WidgetRef ref) {
+    final connectivity = ref.watch(connectivityChangesProvider);
+    final isOnline = connectivity.value?.isOnline ?? false;
```

Comment:
You should not access `value` directly but use `AsyncValue`'s `when()` method: https://pub.dev/documentation/riverpod/latest/riverpod/AsyncValue-class.html

Reply:
If a consumer of an [AsyncValue](https://pub.dev/documentation/riverpod/latest/riverpod/AsyncValue-class.html) does not care about the loading/error state, consider using [value](https://pub.dev/documentation/riverpod/latest/riverpod/AsyncValue/value.html)/valueOrNull to read the state:

Do we really need error and load value here?